js/wizard.js

```diff
@@ -1,5 +1,10 @@
 $(document).ready(function() {
 	
+	//Featherlight lightbox loads when page loads	
+	window.onload = function() {
```

Comment:
I don't think we need to wait for everything to be loaded for this function. Document ready is sufficient. You can remove this code.